### PR TITLE
Pullrequest for two forks

### PR DESCRIPTION
**Pull request by Sickeroni**

### CODE REVIEW

**Review by CvRXX — COMMENTED** [review]
@chieltbest You were working on the clangformat or something right?

File: Lib/Mpl/brigand.hpp [hunk 1]
```diff
@@ -520,6 +520,11 @@ namespace brigand
   template <typename L>
   using as_pair = wrap<L, pair_wrapper>;
 }
+/*********************************************************************
```
Comment (CvRXX) [inline Lib/Mpl/brigand.hpp:523]: Removing unused code is always great but I don't think we should take lwip into a count. What do you think @odinthenerd @chieltbest ?

**Review by CvRXX — CHANGES_REQUESTED** [review]
I think changing things solely because of lwip is wrong. Removing unused things is great but the comments are in my mind unnecessary.

**Comment by CvRXX** [comment]
BTW @odinthenerd and @chieltbest are working on replacing brigand completely with our own mpl so brigand will be out anyways. So for now i suggest that you keep the changes in brigand in your fork and don't push them upstream because they will be gone there soon because of the new mpl

**Comment by Sickeroni** [comment]
closed through some mistakes with seam.hpp

**Pull request closed without merging**